lib/simplifyExpression/basicsSearch/mixedNumber.js

```diff
@@ -0,0 +1,115 @@
+const Negative = require('../../Negative');
+const Node = require('../../node');
+
+// Returns true if `node` is a mixed number
+// e.g. 2 1/2, 19 2/3
+// These will be parsed by mathjs as e.g. 2(1)/(2),
+// which is division with implicit multiplication in the numerator
```

Comment:
Yup, somehow the flow from Mathpix  (latex) -> asciimath -> Athena -> Mathsteps gets us with that representation. Not sure how it changes along the way -- I can dig into that when I have better wifi (on a plane now, lol).